src/main/java/nz/ac/waikato/its/dspace/app/xmlui/aspect/featuredcontent/FeaturedContentController.java

```diff
@@ -0,0 +1,88 @@
+package nz.ac.waikato.its.dspace.app.xmlui.aspect.featuredcontent;
+
+import org.apache.commons.lang3.StringUtils;
+import org.dspace.content.Item;
+import org.dspace.core.ConfigurationManager;
+import org.dspace.core.Context;
+
+import java.io.*;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * Created by jjung on 9/02/16.
+ */
+public class FeaturedContentController {
+
+    static HashMap<String,String> getFeaturedItem(){
+        HashMap<String,String> returnMap = null;
+
+        // configuration file location -> ${DSPACE_HOME}/config/modules/lconz-aspect.cfg
+        String featuredItemFile = ConfigurationManager.getProperty("lconz-aspect", "featured.content.file");
```

Comment:
Please use `[dspace]/config/modules/uow-aspects.cfg` instead, as mentioned in the Google Doc
The name of the property should be `featured-content.datafile`, also as stated in the Google Doc